ArmJoints joints_input
ArmJoints joints_target
---
moveit_msgs/RobotTrajectory trajectory
